DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs

```diff
@@ -14,7 +14,7 @@ public DelegateInfoViewModel(DelegateUserCard delegateUser, IEnumerable<CustomFi
             Id = delegateUser.Id;
             Name = delegateUser.SearchableName;
             CandidateNumber = delegateUser.CandidateNumber;
-            
+

```

Comment:
Empty line?

Reply:
It sections the properties for readability. I assume the change to it is something the auto-formatter did.